cpp-client/README.md

```diff
@@ -207,7 +207,9 @@ Notes
 
 # Building the C++ client on Windows 10 / Windows 11
 
-1. Get Deephaven running by following the instructions here:
+1. To run the client tests (last step in this list), we will need a server.
+   Get Deephaven server ready by following the instructions below.
+   Note the server itself is only supported on linux at this time.
```

Comment:
I agree with the motivation for the change, but maybe we can be even more explicit.... * * * **BOTH** * * * here and in the Ubuntu instructions above. Really tripling-down on the explicitness:

Maybe text like

To actually use Deephaven, for example running these examples and unit tests, you will eventually need to have a server running. If you have an existing server running Deephaven Core, you should be able to connect to that. However, if you don't have one, you can follow the instructions here: https://deephaven.io/core/docs/how-to-guides/launch-build/   .   Note that it is only possible to build a server on Linux. Building a server on Windows is not currently supported.

You can build and install client libraries, tests, and examples without having a server installed. However you will eventually need to connect a server when you want to run them.